tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Core.cs

```diff
@@ -41,6 +41,20 @@ private SecurityCoordinator(Security security, Span span, HttpTransport transpor
         return new SecurityCoordinator(security, span, new(context));
     }
 
+    internal static SecurityCoordinator? TryGetSafe(Security security, Span span)
+    {
+        if (AspNetCoreAvailabilityChecker.IsAspNetCoreAvailable())
+        {
+            var secCoord = GetSecurityCoordinatorImpl(security, span);
+            return secCoord;
+        }
+
+        return null;
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        SecurityCoordinator? GetSecurityCoordinatorImpl(Security securityImpl, Span spanImpl) => TryGet(securityImpl, spanImpl);
```

Comment:
Do we need to define GetSecurityCoordinatorImpl instead of just writing return TryGet(security, span); in line 48?

Reply:
yes because of `[MethodImpl(MethodImplOptions.NoInlining)]` , we wanna make sure the method is not inlined, as it would cause a crash, trying to load a web assembly in a console app execution context